saltlint/rules/NestedDictRule.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Warpnet B.V.
+
+import re
+from saltlint.linter.rule import Rule
+from saltlint.utils import get_rule_skips_from_text
+from saltlint.utils import LANGUAGE_SLS
+
+
+class NestedDictRule(Rule):
+    id = "219"
+    shortdesc = "Nested dictionaries (in context or defaults) should be over-indented"
+    description = "Nested dictionaries (in context or defaults) should be over-indented"
+
+    severity = "HIGH"
+    languages = [LANGUAGE_SLS]
+    tags = ["formatting"]
+    version_added = "develop"
+
+    regex = re.compile(
+        r"^(\s+)-\s+(context|defaults):[^{[\n]*\n^\1\s{0,2}[^-{[\s]*:\s.+",
```

Comment:
I made a mistake in suggesting to change `\s{0,3}` to `\s{0,2}`.  The former is better because is also makes sure at least 2 spaces are used for over-indentation as suggested by the [saltstack documentation](https://docs.saltproject.io/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html#indentation).

So I would suggest to revert back to what your regex originally contained in this part:
```suggestion
        r"^(\s+)-\s+(context|defaults):[^{[\n]*\n^\1\s{0,3}[^-{[\s]*:\s.+",
```

Reply:
ok, I reverted back to 3